### PR TITLE
[DQT] Replace fields list with <SearchableDropdown/>

### PR DESCRIPTION
**Pull request by zaliqarosli**
### Brief summary of changes

@ridz1208 requested we update this as a result of https://github.com/aces/Loris/pull/4463

### CODE REVIEW

File: modules/dataquery/jsx/react.fieldselector.js [hunk 1]
```diff
@@ -358,12 +358,12 @@ var FieldSelector = React.createClass({
                     <div className="form-group col-sm-8 search">
                         <label className="col-sm-12 col-md-2">Instrument:</label>
                         <div className="col-sm-12 col-md-8">
-                            <SelectDropdown
-                                multi={false}
+                              <SearchableDropdown
```
Comment (zaliqarosli) [inline modules/dataquery/jsx/react.fieldselector.js]:
```suggestion
                            <SearchableDropdown
```

File: modules/dataquery/jsx/react.fieldselector.js [hunk 2]
```diff
@@ -358,12 +358,12 @@ var FieldSelector = React.createClass({
...
+                                name="fieldsDropdown"
                                 options={this.state.instruments}
-                                onFieldClick={this.onCategorySelect}
-                                selectedCategory={this.state.selectedCategory}
-                            />
+                                onUserInput={this.onCategorySelect}
+                                placeHolder="Select One"
+                              />
```
Comment (zaliqarosli) [inline modules/dataquery/jsx/react.fieldselector.js]:
```suggestion
                            />
```

**Comment by driusan** [comment]
@HenriRabalais did you test it?

**Comment by HenriRabalais** [comment]
@driusan no but I can do that now!

**Comment by HenriRabalais** [comment]
I'm having trouble with not only this PR, but aces/minor in general. Attempting to access the dashboard, I get the following error:
```
[Fri May 10 14:16:21.887182 2019] [php7:error] [pid 30281] [client 132.216.236.83:36796] PHP Fatal error:  Uncaught DatabaseException: Unknown column 'c.CenterID' in 'where clause' in /var/www/loris/php/libraries/Database.class.inc:686\nStack trace:\n#0 /var/www/loris/php/libraries/Database.class.inc(711): Database->execute(Object(PDOStatement), Array)\n#1 /var/www/loris/php/libraries/Database.class.inc(799): Database->pselect('SELECT COUNT(*)...', Array)\n#2 /var/www/loris/php/libraries/Database.class.inc(927): Database->pselectRow('SELECT COUNT(*)...', Array)\n#3 /var/www/loris/modules/dashboard/php/dashboard.class.inc(311): Database->pselectOne('SELECT COUNT(*)...', Array)\n#4 /var/www/loris/modules/dashboard/php/dashboard.class.inc(82): LORIS\\dashboard\\Dashboard->getTotalRecruitment()\n#5 /var/www/loris/php/libraries/NDB_Form.class.inc(188): LORIS\\dashboard\\Dashboard->setup()\n#6 /var/www/loris/src/Middleware/UserPageDecorationMiddleware.php(171): NDB_Form->handle(Object(Zend\\Diactoros\\ServerRequest))\n#7 /var/www/loris/src/Middleware/PageDecorationMiddleware.php(49): LORIS\\Middleware\\UserPageDecorationMiddleware->proc in /var/www/loris/php/libraries/Database.class.inc on line 686
```

Is this a case of missing a SQL patch?

**Comment by ridz1208** [comment]
@HenriRabalais your problem is that you ran the SQL patches for major, you should have a dump for minor that you restore to when you go backwards.

this specific case you need to undo the centerid to registrationcenterid change

**Comment by HenriRabalais** [comment]
Getting an error on minor when trying to access the DQT:
```
[Fri May 10 15:28:38.726094 2019] [php7:error] [pid 30694] [client 132.216.236.15:41118] PHP Fatal error: 
Uncaught TypeError: Argument 1 passed to NDB_Factory::couchDB() must be of the type string, null given,
called in /var/www/loris/modules/dataquery/php/dataquery.class.inc on line 59 and defined in
/var/www/loris/php/libraries/NDB_Factory.class.inc:238\nStack trace:\n#0
/var/www/loris/modules/dataquery/php/dataquery.class.inc(59): NDB_Factory->couchDB(NULL, NULL,
NULL, NULL, NULL)\n#1 /var/www/loris/php/libraries/NDB_Form.class.inc(188):
LORIS\\dataquery\\Dataquery->setup()\n#2
/var/www/loris/src/Middleware/UserPageDecorationMiddleware.php(171): NDB_Form
>handle(Object(Zend\\Diactoros\\ServerRequest))\n#3
/var/www/loris/src/Middleware/PageDecorationMiddleware.php(49):
LORIS\\Middleware\\UserPageDecorationMiddleware->process(Object(Zend\\Diactoros\\ServerRequest),
Object(LORIS\\dataquery\\Dataquery))\n#4 /var/www/loris/php/libraries/NDB_Page.class.inc(653):
LORIS\\Middleware\\PageDecorationMiddleware->process(Object(Zend\\Diactoros\\ServerRequest),
Object(LORIS\\dataquery\\Dataquery))\n#5 /var/www/loris/php/libraries/Module.class.inc in
/var/www/loris/php/libraries/NDB_Factory.class.inc on line 238, referer: http://132.206.37.67/
```

**Comment by ridz1208** [comment]
@HenriRabalais in you config.xml do you have DQT credentials setup ?

**Comment by zaliqarosli** [comment]
replaced by https://github.com/aces/Loris/pull/4740, closing this PR

**Pull request closed without merging**